src/boot.js

```diff
@@ -1,86 +0,0 @@
-(function(scope) {
```

Comment:
This file was renamed to `targeting.js` since it doesn't actually do any bootstrapping anymore. Bootstrapping wasn't even it's intended purpose before; it just did that because it was the first file being included.